bifunctors.cabal

```diff
@@ -54,42 +39,27 @@ flag tagged
 
 library
   hs-source-dirs: src
-  include-dirs: include
-  includes: bifunctors-common.h
   build-depends:
-    base                >= 4.3   && < 5,
+    base                >= 4.9   && < 5,
     base-orphans        >= 0.8.4 && < 1,
     comonad             >= 5.0.7 && < 6,
     containers          >= 0.2   && < 0.7,
-    template-haskell    >= 2.4   && < 2.20,
+    template-haskell    >= 2.11  && < 2.20,
     th-abstraction      >= 0.4.2.0 && < 0.5,
-    transformers        >= 0.3   && < 0.7
+    transformers        >= 0.5   && < 0.7
 
   if !impl(ghc > 8.2)
     build-depends: transformers-compat >= 0.5 && < 0.8
 
-  if !impl(ghc >= 8.0)
-    build-depends: fail == 4.9.*
-
   if flag(tagged)
     build-depends: tagged >= 0.8.6 && < 1
 
-  if flag(semigroups) && !impl(ghc >= 8.0)
-    build-depends: semigroups >= 0.18.5 && < 1
-
-  if impl(ghc<7.9)
-    hs-source-dirs: old-src/ghc709
-    exposed-modules: Data.Bifunctor
-
   if impl(ghc<8.1)
     hs-source-dirs: old-src/ghc801
     exposed-modules:
       Data.Bifoldable
       Data.Bitraversable
 
-  if impl(ghc>=7.2) && impl(ghc<7.5)
-    build-depends: ghc-prim == 0.2.0.0
-
   exposed-modules:
```

Comment:
The ` Paths_bifunctors` could be removed from `other-modules`, can't it?

Reply:
Yes, you are quite right. I've removed `Paths_bifunctors` from this list.